src/countdown-timer/countdown-timer.tsx

```diff
@@ -144,23 +186,53 @@ export const CountdownTimer = ({
         return offsetY;
     }
 
+    const getAccessibleTimeText = (seconds: number): string => {
+        const { minutes, seconds: secs } = TimeHelper.toMinutesSeconds(seconds);
+
+        const minuteText = minutes === 1 ? "minute" : "minutes";
+        const secondText = secs === 1 ? "second" : "seconds";
+
+        if (minutes > 0 && secs > 0) {
+            return `Time left: ${minutes} ${minuteText} and ${secs} ${secondText}`;
+        } else if (minutes > 0) {
+            return `Time left: ${minutes} ${minuteText}`;
+        } else {
+            return `Time left: ${secs} ${secondText}`;
+        }
+    };
     // =============================================================================
     // RENDER FUNCTION
     // =============================================================================
 
+    const renderInitialAnnouncement = () => (
+        <VisuallyHidden aria-live="polite" aria-atomic="true">
+            {initialAnnouncement}
+        </VisuallyHidden>
```

Comment:
I noticed that in reading mode (where the screen reader navigates through all items), the "Start of..." gets announced as well. Wondering if there's a way to workaround this? Could the initialAnnouncement state be cleared after some time?